README.md

```diff
@@ -1,39 +1,33 @@
 # Katalon Studio action test
 
-This action to run KS test and with your test cases source codes. 
+This action allows you to run KS test and with your test cases source codes. 
 
 ## Inputs
 
-Input your katalon command and apiKey
-refer to more information for using command from https://docs.katalon.com/
+Input your katalon studio version number, katalon command, and apiKey
 
-**Required** the katalon commands , apikey and test cases files 
+**Required** the katalon studio version number, katalon commands , apikey and test cases files 
 
 
 This is the example to using github action <br>
 Please change to the latest github action version and your Input. <br>
 
-Setup API Key using Secret name :  KATALON_API_NAME
-
-Katalon Github Action Marketplace link :  https://github.com/marketplace/actions/katalon-studio
-
+Setup API Key using Secret name :  API_KEY
```

Comment:
https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets